src/Memento.jl

```diff
@@ -38,6 +41,7 @@ include("filters.jl")
 include("formatters.jl")
 include("handlers.jl")
 include("loggers.jl")
+include("deprecates.jl")
```

Comment:
I think either the file name "deprecated.jl" or "deprecations.jl" would read better.

Reply:
"deprecates" is what a lot of the stats libraries use, but I have no strong preference.